README.md

```diff
@@ -1,51 +1,164 @@
-# Polyfill service
+Polyfill service
+=====================
+
+Makes web development less frustrating by selectively polyfilling just what the browser needs. Use it on your own site, or as a service.
+
+For usage information see the [hosted service](polyfill-service), which formats and displays the service API documentation located in the [docs](docs/) folder.
 
 [![Build
-Status](https://circleci.com/gh/Financial-Times/polyfill-service.svg?&style=shield&circle-token=357956eb8e6bea4ae9cca8f07918b7d0851a62d1)](https://circleci.com/gh/Financial-Times/polyfill-service)
+Status](https://circleci.com/gh/Financial-Times/polyfill-service.svg?&style=shield&circle-token=357956eb8e6bea4ae9cca8f07918b7d0851a62d1)][ci]
+[![CC0 licensed](https://img.shields.io/badge/license-CC0-blue.svg)][license]
+
+
+Table Of Contents
+-----------------
+
+  * [Requirements](#requirements)
+  * [Running Locally](#running-locally)
+  * [Configuration](#configuration)
+  * [Testing](#testing)
+	* [Release Process](#release-process)
+  * [Deployment](#deployment)
+	* [Publishing](#publishing)
+  * [Monitoring](#monitoring)
+  * [Library API Reference](#library-api-reference)
+  * [License](#license)
+
+
+Requirements
+------------
+
+Running Origami Build Service requires a few tools:
+  * [Git]: For downloading the source code
+  * [Node.js] 5.x and [npm] 3.x: For installing the dependencies and running the application
+	* [Grunt] 0.1.x: Used for automating tasks such as testing
+
+Running Locally
+---------------
+
+Clone the project to our system:
+```sh
+git clone git@github.com:Financial-Times/polyfill-service.git
+```
+
+Change into the project directory:
+```sh
+cd polyfill-service
+```
+
+Install the dependencies:
+```sh
+npm install
+```
+
+Build the polyfill sources and start the server, rebuiling and restarting whenever any changes are made to the project:
+```sh
+grunt dev
```

Comment:
`dev` is the name of the grunt task. Should we list out the grunt tasks somewhere?